### PR TITLE
Add Text entity

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

CleanShot 2022-11-24 at 14 11 57

CleanShot 2022-11-24 at 14 12 04

Parent PR in Core: <https://github.com/home-assistant/core/pull/79454>

ℹ️ This PR renamed `hui-text-entity-row` -> `hui-simple-entity-row` to free up the name for use with this entity.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/entity-rows/hui-text-entity-row.ts [hunk 1]
```diff
@@ -48,22 +45,53 @@ class HuiTextEntityRow extends LitElement implements LovelaceRow {
...
+          .label=${this._config.name || computeStateName(stateObj)}
+          .disabled=${stateObj.state === UNAVAILABLE}
+          .value=${stateObj.state}
+          .minlength=${stateObj.attributes.min}
+          .maxlength=${stateObj.attributes.max}
+          .autoValidate=${stateObj.attributes.pattern}
+          .pattern=${stateObj.attributes.pattern}
+          .type=${stateObj.attributes.mode}
+          @change=${this._valueChanged}
+          placeholder="(empty value)"
```
Comment (piitaya) [inline src/panels/lovelace/entity-rows/hui-text-entity-row.ts:63]: We must translate this (for `input_text` too). We can do that in another PR.

**Review by piitaya — DISMISSED** [review]

File: src/panels/lovelace/entity-rows/hui-text-entity-row.ts (hunk 1 above)
Comment (frenck) [reply src/panels/lovelace/entity-rows/hui-text-entity-row.ts]: Agreed, added it.

**Review by piitaya — APPROVED** [review]

**Comment by frenck** [comment]
Backend PRs have been merged
